css/index.css

```diff
@@ -143,4 +226,46 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* BOXES */
+
```

Comment:
Great use of developer comments within your code. If a developer has to ever come behind you to look at your code, they can better understand your thought process.